mod_conf_sql.c

```diff
@@ -157,6 +157,12 @@ static int sqlconf_parse_uri_db(char **uri) {
   *tmp = '\0';
   sqlconf_db.database = pstrdup(sqlconf_pool, *uri);
 
+  int i;
+  int n=strlen(sqlconf_db.server);
+
+  // replace \ with /
+  for(i=0;i<=n;i++) if(sqlconf_db.server[i]=='\\') sqlconf_db.server[i]='/';
```

Comment:
when start proftpd with sql:// uri... and if you transmit absolute path to the sqlite file with / in file path... you must use another character than / in the path...

exemple :
proftpd -c sql://nouser:nopass@\path\to\sqlitefile.db/db:SQlite/ctxt:ftpctxt/conf:ftpconf/map:ftpmap